api/src/opentrons/protocol_engine/commands/movement_common.py

```diff
@@ -115,6 +115,22 @@ class StallOrCollisionError(ErrorOccurrence):
     detail: str = ErrorCodes.STALL_OR_COLLISION_DETECTED.value.detail
 
 
+class FlexStackerStallOrCollisionError(ErrorOccurrence):
+    """Returned when the machine detects that axis encoders are reading a different position than expected.
```

Comment:
Stacker doesn't have any encoders so I think you can leave it as:
```suggestion
    """Returned when the motor driver detects a stall."""
```